docs/guide/third_party.rst

```diff
@@ -0,0 +1,54 @@
+Third-party plugins
+===================
+This page explains how ``napari-matplotlib`` can be used within third party plugins.
+
+``napari-matplotlib`` provides a ready-to-go widget with a Matplotlib toolbar and figure to third party plugin developers
+This widget is customised to match the theme of the main napari window.
+
+The widget can be found at `napari_matplotlib.base.NapariMPLWidget`.
+This class inherits from ``qtpy.QtWidgets.QWidget``.
```

Comment:
Was this your intersphinx issue? S'a shame we can't link to them. Is it worth it to hardcode the link, or does that way madness lie?

Reply:
I was hoping I could link to `qtpy`, but it turns out this isn't possible. So I've changed this to link out to `PyQT6`.

Reply:
I've just tried to get intersphinx linking with either PyQT6 or PySide2 but to no avail, so I've just stuck in a manual link for now.